src/main/java/com/betadevels/onlineshopping/action/subscription/CreateSubscriptionsAction.java

```diff
@@ -80,6 +79,9 @@ public SubscriptionListResponse invoke()
 					Cart cart = cartOptional.get();
 					LocalDateTime now = LocalDateTime.now();
 
+					List<Subscription> userSubscriptions = subscriptionHibernateDAO.getSubscriptionsOfUser(page,size,cart.getCustomer());
```

Comment:
We wont be needing this, but still where are you setting the values of page and size variables?